poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
```

Comment:
You need to run `poetry lock` because i'm getting an error on your branch when i try to build the container:

```
 => ERROR [app 9/9] RUN poetry install                                                   1.0s
------                                                                                        
 > [app 9/9] RUN poetry install:                                                              
0.558 Creating virtualenv hushline-9TtSrW0h-py3.12 in /root/.cache/pypoetry/virtualenvs       
0.818 Installing dependencies from lock file                                                  
0.917 
0.917 pyproject.toml changed significantly since poetry.lock was last generated. Run `poetry lock [--no-update]` to fix the lock file.
------
failed to solve: process "/bin/sh -c poetry install" did not complete successfully: exit code: 1
```